lib/Sema/CSDiagnostics.cpp

```diff
@@ -3399,6 +3402,38 @@ bool MissingMemberFailure::diagnoseForDynamicCallable() const {
   return false;
 }
 
+bool MissingMemberFailure::diagnoseForDefaultAnyInCollection() const {
+  auto &cs = getConstraintSystem();
+  auto *expr = getAsExpr(getAnchor());
+  auto *parentExpr = cs.getParentExpr(expr);
+  auto &solution = getSolution();
+
+  if (parentExpr && isa<UnresolvedMemberExpr>(expr)) {
+    auto parentType = resolveType(getType(parentExpr));
```

Comment:
Note: `getType` already resolves a type so you don't have to use `resolveType` here.